README.md

```diff
@@ -1,222 +1,134 @@
-* React JS along with Node JS
-
-**What if I don't know any of the mentioned technologies, but I am ready to learn?**
-
-That's completely fine. Just ping me on [LinkedIn](https://www.linkedin.com/in/praveentech/) and I am more than happy to teach you!
-
-**Where's the code? Is anything available?**
-
-Nothing is there other than this ReadMe document. Please hold your horses until the launch event on 30th.
-
-**What will you get in return by contributing to this project?**
-
-Goodies, swags, internships! For internships in the UK, please talk to me.
-
-**When are we starting the project?**
-
-The project should have everything to be getting started by the New Year's Eve!
-
-**What's the process for contributing to this project?**
-
-Please check the contributing guidelines of my previous project, and it's going to be almost same.
-
-**Who are the project maintainers and mentors for this project?**
-
-* Praveen Kumar Purushothaman ([Website](https://praveen.science/) &#1421; [LinkedIn](https://www.linkedin.com/in/praveentech/) &#1421; [YouTube](https://www.youtube.com/praveenscience?sub_confirmation=1))
-* Manju Ramachandran ([LinkedIn](https://www.linkedin.com/in/manjuramachandran10/))
-* Saloni Sawarkar ([LinkedIn](https://www.linkedin.com/in/saloni-sawarkar/))
-
-## Our Contributors ✨🎉
-
-<a href="https://github.com/praveenscience/Internship-LMS-FrontEnd/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=praveenscience/Internship-LMS-FrontEnd" />
-</a>
-
-
--->
-
-# DevTernShip: Internship OnBoarding Platform
  
+ ## DevTernShip: Internship OnBoarding Platform 
 
 ![](https://user-images.githubusercontent.com/1830380/111150758-65810c00-8586-11eb-8941-7985fb66f23c.png)
 
-# BADGES
 
+[![Issues](https://img.shields.io/github/issues/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues)
+[![PRs](https://img.shields.io/github/issues-pr/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/pulls)
+[![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+[![Forks](https://img.shields.io/github/forks/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Watchers](https://img.shields.io/github/watchers/praveenscience/foss-events?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
 
 # Introduction
 
- 
-
-
- # Motivation
-
-
-
+ We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
```

Comment:
Why space in front? Let's rephrase this for a better intro or should we give it to someone else?